.gitignore

```diff
@@ -8,4 +8,8 @@
 /yarn-error.log
 
 .byebug_history
-/coverage
\ No newline at end of file
+/coverage/public/packs
+/public/packs-test
```

Comment:
add `/public/packs` in gitignore, and you have to remove it from git, `$ git rm -rf /public/packs`